cypher-shell/src/main/java/org/neo4j/shell/commands/Param.java

```diff
@@ -56,13 +58,26 @@ public String getHelp() {
 
     @Override
     public void execute(@Nonnull final String argString) throws CommandException {
-        Matcher m = argPattern.matcher(argString);
+        Matcher alphanumericMatcher = argPattern.matcher(argString);
 
-        if (!m.matches()) {
-            throw new CommandException(AnsiFormattedText.from("Incorrect number of arguments.\nusage: ")
-                    .bold().append(COMMAND_NAME).boldOff().append(" ").append(getUsage()));
+        if (!alphanumericMatcher.matches()) {
```

Comment:
I think this method's readability can be improved now. What do you think about this?

```java
Matcher matcher = argPattern.matcher(argString);

if (!matcher.matches() && argString.trim().startsWith("`")) {
    matcher = backtickPattern.matcher(argString);
}

if (matcher.matches() && matcher.group("key").length() > 2) {
    variableHolder.set(matcher.group("key"), matcher.group("value"));
} else {
    throwError();
}
```